README.md

```diff
@@ -5,13 +5,45 @@ List=[1,2,3,4]
 
 
 - Question 1: Write a function that takes two numbers and return their sum 
+     def func(x,y):
+     sum=x+y
+     print(sum) ask about input
+     func(3,2)
+     
 - Question 2: Prints all the elements in the list using a for loop
+    for x in List:
+     print (x) 
 - Question 3: Write a Python program to sum all the items in the list 
+     print(sum(list))
 - Question 4: Write a Python program to get the largest number from the list
+     print(max(list))
+     
 - Question 5: write a function that take a list and a number then return a new partial list starting from index 0 to index "number"
+
+def split(x:list, i:int):
+return x[:n]
+
 - Question 6: Loop through the letters in the string: "Tuwaiq_Academy
+     string_name = "Tuwaiq_Academy"
+     for element in string_name:
+        print(element, end='  ')
+     print("\n")
+     
 - Question 7: Consider this ``` list = ["Python", "C++", "Java"] ``` Exit the loop when x is equal to "C++"
```

Comment:
list = ["Python", "C++", "Java"] 
for x in list:
if x == "C++":
break
print(x)